driver/tests/mpi/run_save_and_load_restart.sh

```diff
@@ -3,9 +3,10 @@ set -e
 MPIRUN_CALL=${MPIRUN_CALL:-mpirun -n 6}
 cp examples/configs/baroclinic_c12_write_restart.yaml baroclinic_c12_write_restart.yaml
 cp examples/configs/baroclinic_c12_write_restart.yaml baroclinic_c12_run_two_steps.yaml
-sed -i 's/seconds: 225/seconds: 450/' baroclinic_c12_run_two_steps.yaml
-sed -i 's/save_restart: true/save_restart: false/' baroclinic_c12_run_two_steps.yaml
-sed -i 's/path: "output.zarr"/path: "run_two_steps_output.zarr"/' baroclinic_c12_run_two_steps.yaml
+sed -i.bak 's/seconds: 225/seconds: 450/' baroclinic_c12_run_two_steps.yaml
+sed -i.bak 's/save_restart: true/save_restart: false/' baroclinic_c12_run_two_steps.yaml
+sed -i.bak 's/path: "output.zarr"/path: "run_two_steps_output.zarr"/' baroclinic_c12_run_two_steps.yaml
+rm *.bak
```

Comment:
What was here didn't work on BSD sed, so I changed it to be compatible in both cases.